model/contributor.py

```diff
@@ -75,6 +75,10 @@ class Contributor(Base):
     PHOTOGRAPHER_ROLE = u"Photographer"
     TRANSLATOR_ROLE = u"Translator"
     ILLUSTRATOR_ROLE = u"Illustrator"
+    LETTERER_ROLE = u"Letterer"
+    PENCILER_ROLE = u"Penciler"
+    COLORIST_ROLE = u"Colorist"
+    INKER_ROLE = u"Inker"
```

Comment:
The test failures are because you added role types without also adding [the corresponding MARC codes](https://www.loc.gov/marc/relators/relaterm.html) to MARC_ROLE_CODES. The only one I can find a direct analog for is Colorist, which has the role code "clr".

[In this comment,](https://github.com/readium/webpub-manifest/issues/26#issuecomment-479429879) @HadrienGardeur recommends treating the other roles as simply "contributor". So use the MARC code "ctb" for the other three.